README.md

```diff
@@ -53,16 +53,38 @@ which is used for local run/debug, because the integration tests will remove all
             - Cart total amount should be > 0
             - Shipping address should be set
         - Required fields for the CTP payment:
-            - amountPlanned
-            - cancelUrl
-            - successUrl
-            - paymentMethodInfo needs to be set like this:
+            - `amountPlanned`
+            - `cancelUrl` (custom field)
```

Comment:
Normally we call this url "errorUrl"

Reply:
normally (in Payone service) we have both `errorUrl` and `cancerUrl`, but Paypal Plus requires only `cancelUrl`